symphony-bdk-spring/symphony-bdk-app-spring-boot-starter/src/main/java/com/symphony/bdk/app/spring/config/BdkExtAppSecurityConfig.java

```diff
@@ -16,7 +15,6 @@
 public class BdkExtAppSecurityConfig {
 
   @Bean
-  @ConditionalOnMissingBean
```

Comment:
(for the "why" a Spring Boot app can have multiple `WebMvcConfigurer`, see [here](https://github.com/spring-projects/spring-framework/blob/4b8ce04bdc45c0e523a2b2e140cdc6b4f8db858a/spring-webmvc/src/main/java/org/springframework/web/servlet/config/annotation/DelegatingWebMvcConfiguration.java#L45))

Reply:
That's why it was not necessary to annotate this bean with `@ConditionalOnMissingBean`

Reply:
We could probably have a test for that, that would fail before and would be fixed now right?
(like starting a context with a test controller and checking that cors header are there)

Reply:
I used to think about it but it seems the `corsConfigurer` bean is created even with the `@ConditionalOnMissingBean`